tools/pot/docs/InstallationGuide.md

```diff
@@ -18,50 +18,3 @@ The simplest way to get the Post-training Optimization Tool and OpenVINO&trade;
 3. To install POT and other OpenVINO&trade; developer tools, run `pip install openvino-dev`.
 
 Now the Post-training Optimization Tool is available in the command line by the `pot` alias. To verify it, run `pot -h`.
-
```

Comment:
This document contains a phrase like "There are two ways how to install the POT on your system:" followed by the list of options. But essentially now there is only one option. I suggest removing or rephrasing this.

Reply:
Thanks! This was a point I wanted to discuss, do we want to include the custom package builds option or mention it anywhere in this doc?